src/main/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder.java

```diff
@@ -282,17 +283,27 @@ final void updateChildren(final TaskListener listener) throws IOException, Inter
                             new Object[]{getFullName(), orphaned.keySet()});
                 }
                 Collection<I> forRemoval = orphanedItems(orphaned.values(), listener);
+
+                List<IOException> deletingExceptions = new LinkedList<>();
                 for (I existing : orphaned.values()) {
                     if (forRemoval.contains(existing)) {
                         if (LOGGER.isLoggable(Level.FINE)) {
                             LOGGER.log(Level.FINE, "{0}: deleting {1}", new Object[]{getFullName(), existing});
                         }
-                        existing.delete();
-                        // super.onDeleted handles removal from items
+                        try {
+                            existing.delete();
+                            // super.onDeleted handles removal from items
+                        } catch (IOException x) { // InterruptedException is still propagated
```

Comment:
why do we want to stop with something different than IOException?

Reply:
The other possibility is InterruptedException and it seems weird to continue if that's the case.